CMakeLists.txt

```diff
@@ -293,6 +293,7 @@ else (NOT MACOSX)
   set (RT_LIBRARY "")
 endif (NOT MACOSX)
 
+
```

Comment:
Just for tidiness, this file could be reverted to its previous state (only blank lines, no need to put in the PR).